### DISCUSSION TITLE
Accedere da remoto senza ip fisso

### DISCUSSION THREAD

**Discussion by giacomo-git** [indomus/forum#1168] [HUB: Home Assistant]
Ciao
Attualmente accedo da remoto al mio ha su docker con caddy e duckdns. 
Il mio provider attuale Fastweb mi fornisce ip pubblico statico
Sto valutando un offerta da Sky wifi che però non fornisce ip pubblico statico.
La domanda è sarebbe possibile accedere cmq da remoto?
Leggendo le guide mi sembra sia possibile gestire duckdns aggiornando dinamicamente l'IP....
Grazie

**Comment by vebto53** [comment]
> Ciao Attualmente accedo da remoto al mio ha su docker con caddy e duckdns. Il mio provider attuale Fastweb mi fornisce ip pubblico statico Sto valutando un offerta da Sky wifi che però non fornisce ip pubblico statico. La domanda è sarebbe possibile accedere cmq da remoto? Leggendo le guide mi sembra sia possibile gestire duckdns aggiornando dinamicamente l'IP.... Grazie

Io personalmente uso NO-IP che aggiorna automaticamente i DNS (ddns dynamic dns)di modo che tu puoi utilizzare un nome.ddns.net al posto dell'IP. Non so come, ma duckdns fa la stessa cosa. L'unico problema con NO-IP è che ogni meseti avvisano e tu ti colleghi e rinnovi il tuo ddns, se no paghi. Ciao

**Comment by snakuzzo** [comment]
Sì...sostanzialmente ti basta avere un IP pubblico (che sia statico o dinamico non ha importanza). 
L'aggiornamento dell'associazione FQDN + IP la si fa con una semplice chiamata REST.
Hai detto che sei su docker... per fare questo puoi utilizzare un'immagine docker ad hoc ([QUI](https://hub.docker.com/r/linuxserver/duckdns/))  oppure puoi evitare e fare un package direttamente su HA ([QUI](https://indomus.it/guide/aggiornare-il-proprio-record-su-duckdns-tramite-automazione-home-assistant/) la nostra guida).

Ad ogni modo, non so come funzioni Sky WIFI, ma assicurati che ti diano un IP pubblico

**Comment by giacomo-git** [comment]
interessante, grazie
